### DISCUSSION TITLE
Keyboard shortcut to activate Expression Editor

### DISCUSSION THREAD

**Discussion by steve-james-hcc** [TabularEditor/TabularEditor3#1350] [🙋♀️TE3 Q&A] [answered]
I've searched the docs, this forum, and the Keyboard section of Preferences, but can't find a keyboard shortcut for activating the Expression Editor when browsing the TOM Explorer. Is there one?

**Comment by corycundy** [comment]
Steve, I would love to know the same. 

Actually as an enhancement, I think it would be great to have the cursor focus go into the Expression Editor when using Alt Left Arrow and Alt Right Arrow when the object is editable. For example, when using Alt Left and Right to go between measures, I naturally want to hit Alt Left Arrow and start editing the DAX, but the TOM has the focus. This obviously doesn't work and wouldn't need to do anything if the Alt Left / Right moves to an non-editable object such as a Table or Column.

Lastly, it would be mimicking the double click nature vs. single click in the TOM. I think the double click (which moves the cursor into the Expression Editor) would be more desirable for most people.

**Comment by mlonsk** [comment, marked as answer]
Hi @steve-james-hcc 

There is actually a Keyboard setting for it in the Preferences. It is called View.QuickEditor and assigning a shortcut to it will let you navigate to the Expression Editor.:

![image](https://github.com/user-attachments/assets/c16ce894-c2f2-4cb0-bb30-303f57247869)

@corycundy does setting this shortcut help your use case as well?

**Reply by steve-james-hcc** [reply]
That's great, many thanks!

**Reply by corycundy** [reply]
@mlonsk, this is certainly helpful, is a good workaround to @steve-james-hcc's question. It alleviates the need from going back to the mouse. While very minor, it would still be nice to have the Editor.QuickBack and Editor.QuickForward place focus to the Expression Editor as well. This would be similar to how double clicking a measure does this vs. single clicking.

I appreciate the View.QuickEditor tip! I will certainly be using this.

**Reply by mlonsk** [reply]
Hi @corycundy 
Thank you for explaning your request further; I can definitely see the need for the focus to switch/stay in the Expression Editor when using those shortcuts. 
Thank you for the suggestion, and we will keep you updated.
